roles/aap_setup_install/vars/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Set the https port used for validating services are up
+__aap_setup_inst_controller_port: "{{ aap_setup_prep_inv_vars.automationcontroller.nginx_https_port | default('8443') }}"
```

Comment:
is there a reason why this is in vars instead of defaults like all the other vars?

Reply:
These values are not expected to be overridden/passed in by the playbook calling the setup install role.  The ports are only changed if the `nginx_https_port` is set in the `aap_setup_prep_inv_vars`. Other wise the default port numbers are used.

Reply:
This brings up another question. Currently the default ports are the TLS enabled ports. Should these point to the http ports and allow the redirect point to the correct TLS enabled port?

Reply:
but why wouldn't they be expected to be over written? I think a lot of people might not want to use the default ports the container installer uses. This is also related to your TLS enabled port question.

Reply:
To set the ports nginx uses during the install, the vars `nginx_http_port` and `nginx_https_port` need to be set in the `inventory.yml`.  We can set these vars in the `inventory.yml` during the prep step by adding the vars to a host group when configuring `aap_setup_prep_inv_vars`. 

One key difference with the containerized installer is that you can run all the services on a single host. By default all the services are configured to listen on non-standard ports.  Because of this, we need to be able to determine if `nginx_http_port` or `nginx_https_port` are set. If they are set then we need to use the values when validating the servies are running or use the default ports below.

| App | HTTP | HTTPS |
|-----|-------|--------|
| Controller | 8080 | 8443 |
| Hub | 8081 | 8444 |
| EDA | 8082 | 8445 |

There are additional complications when installing on a single node. You can not set `nginx_http_port` or  `nginx_https_port` as it will assign the same port for all services. This can be worked around with multiple host entires in the /etc/hosts for the same host. Also the default ports changed between the current version of the installer and previous versions.

Reply:
Yes I know which is why I think we should make this be an expected variable from the user to set what port they want. I will double check my container inventory because I haven't messed with it much since I configured it in the beta testing.

Reply:
I do understand what you mean about the nginx_http(s)_port var and if it's the same host it will mess stuff up. I just wonder if we should make a check that verifies that if they set port it cannot have the same hosts... I would be interested in hearing what others think about how we should solve/set this

Reply:
@sean-m-sullivan @Tompage1994 @ericzolf

Reply:
I think as is, is fine, we could add the feature in the future if we add the other logic around setting multiple hosts, but I believe the containerized is supposed to be single node by design.